ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml

```diff
@@ -58,6 +58,10 @@ releases:
       product: ocp
       stream: nightly
       version: "4.12"
+  stable-4-11:
+    integration:
+      name: "4.11"
+      namespace: ocp
```

Comment:
iiuc this doesn't pick stable 4.11 versions (4.11.z)
but rather you'll get ``ci`` stream, which is what's merged to the ``release-4.11`` branches (e.g. ``4.11.0-0.ci.test-2022-11-23-210525``)

Reply:
I have decided to remove this as I am going with advice you give later on about using the default.
Though I am curious about this because  I notice that the config here uses similar terminology

https://github.com/openshift/release/blob/master/ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml#L170-L177

Maybe I need a better understanding of release streams, I will chat with you about this I think.

Reply:
Yes, we probably did the same mistake there
As for the documentation for those streams: https://docs.ci.openshift.org/docs/architecture/ci-operator/#testing-with-an-existing-openshift-release